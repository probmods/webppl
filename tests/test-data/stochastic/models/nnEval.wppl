var net = nn.constantparams([1]);

var model = function() {
  return sample(Gaussian({mu: -4, sigma: 1}), {guide() {
    var mu = T.get(nnEval(net), 0);
    return Gaussian({mu, sigma: 1});
  }});
};
