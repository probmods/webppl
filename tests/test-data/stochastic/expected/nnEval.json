{
  "mean": -4,
  "std": 1
}
